src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -44,7 +44,7 @@ public class ProviderInfo
         public string Name { get; }
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.8.137', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```